app/main.py

```diff
@@ -1,3 +1,73 @@
+from __future__ import annotations
```

Comment:
The `from __future__ import annotations` statement is unnecessary in this context unless you are using forward references in type hints. If not needed, it can be removed.